### PR TITLE
fix(LinkSelector): initial click triggers twice

### PR DESCRIPTION
**Pull request by fulopdaniel**
onClick and onFocus did the same thing, so initial click on a link item resulted in toggling it twice => essentially canceling the action.

Removed the onFocus handlers, and changed the arrow buttons to be <button> elements so the link chooser is still accessible.

https://www.loom.com/share/309c34b38c6c4a94a84cd1177ac6a848

### CODE REVIEW

**Review by ragi96 — COMMENTED** [review]
Tests that are failing are related to your changes

File: packages/guideline-blocks-settings/src/components/Link/LinkSelector/DocumentLink.tsx [hunk 1]
```diff
@@ -39,29 +39,27 @@ export const DocumentLink = ({
             <button
```
Comment (ragi96) [inline packages/guideline-blocks-settings/src/components/Link/LinkSelector/DocumentLink.tsx:39]: Add a changeset

File: packages/guideline-blocks-settings/src/components/Link/LinkSelector/PageLinks.tsx [hunk 2]
```diff
@@ -45,8 +45,8 @@ export const PageLinks = ({
 
     if (isLoading) {
         return (
-            <div className="tw-flex tw-justify-center tw-p-4">
-                <LoadingCircle />
+            <div className="tw-flex tw-justify-center tw-h-10 tw-items-center">
+                <LoadingCircle size={LoadingCircleSize.Small} />
             </div>
```
Comment (ragi96) [inline packages/guideline-blocks-settings/src/components/Link/LinkSelector/PageLinks.tsx:50]: This is duplicated with the one here: `packages/guideline-blocks-settings/src/components/Link/LinkSelector/DocumentLinks.tsx` can we have it as an own component and just reuse it?

File: .changeset/tiny-wasps-arrive.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/guideline-blocks-settings": patch
+---
+
+fix(LinkSelector): bug fixes and qol changes
```
Comment (ragi96) [inline .changeset/tiny-wasps-arrive.md:5]:
```suggestion
fix(LinkSelector): initial click triggered twice
```
What do you think? 
fix(): bug fix is pretty odd as it says both times the same

Comment (fulopdaniel) [reply .changeset/tiny-wasps-arrive.md]: Sure!

**Review by ragi96 — APPROVED** [review]

**Review by anxobotana — APPROVED** [review]

**Pull request merged**